lib/libzfs/libzfs_import.c

```diff
@@ -1687,18 +1802,16 @@ zpool_find_import_scan(libzfs_handle_t *hdl, pthread_mutex_t *lock,
 	rdsk_node_t *slice;
 	void *cookie;
 	int i, error;
+	struct stat sbuf;
```

Comment:
You might want to move this declaration in to the `for ()` block to make the scope clearer.